src/core/proxy.js

```diff
@@ -45,6 +45,7 @@ const requestHandler = async (request, proxy, overrides = {}) => {
     } catch (error) {
         console.error("proxy requestHandler error caught:", error)
         await request.abort();
+        throw error;
```

Comment:
I don't think this gets caught anywhere. what's the point?

Reply:
it will get caught by useProxy function